BEIMA.Backend.Test/StorageService/AzureStorageProviderTest.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using BEIMA.Backend.StorageService;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace BEIMA.Backend.Test.StorageService
+{
+    [TestFixture]
+    public class AzureStorageTest : UnitTestBase
+    {
+        private readonly IStorageProvider _storage;
+
+        public AzureStorageTest()
```

Comment:
This looks like it should be put up in a OneTimeSetUp or SetUp method.

Reply:
Its so I can keep it as `readonly IStorageProvider` instead of `IStorageProvider?` not sure if that matters or not so I'll defer to putting it in a OneTimeSetup

Reply:
Actually I can just move it to a function call.